contrib/core-contract-tests/deployments/default.simnet-plan.yaml

```diff
@@ -0,0 +1,87 @@
+    - name: wallet_1
+      address: ST1SJ3DTE5DN7X54YDH5D64R3BCB6A2AG2ZQ8YPD5
+      balance: "100000000000000"
+    - name: wallet_2
+      address: ST2CY5V39NHDPWSXMW9QDT3HC3GD6Q6XX4CFRK9AG
+      balance: "100000000000000"
+    - name: wallet_3
+      address: ST2JHG361ZXG51QTKY2NQCVBPPRRE2KZB1HR05NNC
+      balance: "100000000000000"
+    - name: wallet_4
+      address: ST2NEB84ASENDXKYGJPQW86YXQCEFEX2ZQPG87ND
+      balance: "100000000000000"
+    - name: wallet_5
+      address: ST2REHHS5J3CERCRBEPMGH7921Q6PYKAADT7JP2VB
+      balance: "100000000000000"
+    - name: wallet_6
+      address: ST3AM1A56AK2C1XAFJ4115ZSV26EB49BVQ10MGCS0
+      balance: "100000000000000"
+    - name: wallet_7
+      address: ST3PF13W7Z0RRM42A8VZRVFQ75SV1K26RXEP8YGKJ
+      balance: "100000000000000"
+    - name: wallet_8
+      address: ST3NBRSFKX28FQ2ZJ1MAKX58HKHSDGNV5N7R21XCP
+      balance: "100000000000000"
+    - name: wallet_9
+      address: STNHKEPYEPJ8ET55ZZ0M5A34J0R3N5FM2CMMMAZ6
+      balance: "100000000000000"
+  contracts:
+    - costs
+    - pox
+    - pox-2
+    - pox-3
+    - pox-4
+    - lockup
+    - costs-2
+    - costs-3
+    - cost-voting
+    - bns
+plan:
+  batches:
+    - id: 0
+      transactions:
+        - emulated-contract-publish:
+            contract-name: bns
+            emulated-sender: ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM
+            path: "../../stackslib/src/chainstate/stacks/boot/bns.clar"
+            clarity-version: 2
+        - emulated-contract-publish:
+            contract-name: bns_test
+            emulated-sender: ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM
+            path: "./tests/bns_test.clar"
+            clarity-version: 2
+        - emulated-contract-publish:
+            contract-name: pox-4
+            emulated-sender: ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM
+            path: "../../stackslib/src/chainstate/stacks/boot/pox-4.clar"
+            clarity-version: 2
+        - emulated-contract-publish:
+            contract-name: pox-mainnet
+            emulated-sender: ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM
+            path: "../../stackslib/src/chainstate/stacks/boot/pox-mainnet.clar"
+            clarity-version: 2
+        - emulated-contract-publish:
+            contract-name: pox_4_test
+            emulated-sender: ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM
+            path: "./tests/pox_4_test.clar"
+            clarity-version: 2
+        - emulated-contract-publish:
+            contract-name: signers
+            emulated-sender: ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM
+            path: "../../stackslib/src/chainstate/stacks/boot/signers.clar"
+            clarity-version: 2
+        - emulated-contract-publish:
+            contract-name: signers-voting
+            emulated-sender: ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM
+            path: "../../stackslib/src/chainstate/stacks/boot/signers-voting.clar"
+            clarity-version: 2
+      epoch: "2.4"
```

Comment:
I guess we need the this and switch to `3.0` on demand in the tests, right?

Reply:
@moodmosaic There a multiple things to consider here;
1. the deployment plan file is just saved with the recent version of the SDK, but in terms of contract deployment, it reflects the clarinet.toml, if the epoch has to be changed, it should probably happen in an other PR to keep this one just a dependency update PR
2. There are a few other things to changes in the clarinet.toml, especially regarding epochs (ie: bns wasn't deployed in 2.4)
3. if the purpose really is to test boot contracts as such, maybe the tests should call the boot contracts deployed by clarinet (as in this example) https://github.com/hugocaillard/pox-testing-sample. Especially for the pox contracts, since calling a custom pox contract won't actually lock the stx